src/cow-react/modules/limitOrders/utils/getFilledAmounts.ts

```diff
@@ -0,0 +1,78 @@
+import { ParsedOrder } from '@cow/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList'
```

Comment:
I see why stays in the module of limit orders. This is specific to this ParsedOrders which is for now only in limit orders module

I think in the future we might want to move some of these things to some orders module that is independent. We might have swap, limit orders, twap, classical trading UI, etc and in principle all this things are the same for all